src/component/child/index.js

```diff
@@ -221,6 +225,34 @@ export class ChildComponent extends BaseComponent {
     }
 
 
+    /* Auto Resizing
+      --------------
+      Checks for change in dimensions of content inside an iframe and calls a resize() if required
+     */
+
+    watchForResize() {
+        var elm = document.body;
+        var lastWidth = elm.scrollWidth, newWidth;
+        var lastHeight = elm.scrollHeight, newHeight;
+
+        setInterval(() => {
+            newWidth = elm.scrollWidth;
+            newHeight = elm.scrollHeight;
+            // Dimensions changed if this condition is true
+            if (lastHeight != newHeight || lastWidth != newWidth) {
+                var newDimensions = {
```

Comment:
Can you please use `let` rather than `var`? Running `gulp lint` should show any other issues like this :)